go/vt/servenv/http.go

```diff
@@ -0,0 +1,39 @@
+package servenv
+
+import (
+	"errors"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/http/pprof"
+)
+
+var mux = http.NewServeMux()
+
+func HTTPHandle(pattern string, handler http.Handler) {
+	mux.Handle(pattern, handler)
```

Comment:
nit: we should add comments to exported methods

Reply:
Done!